TextField/TextField.jsx

```diff
@@ -228,6 +234,7 @@ class TextField extends Component {
       <TextFieldInput
         {...props}
         className={className}
+        outerStyle={outerStyle}
```

Comment:
delete this, outer style should only apply to outermost container

Reply:
In that case this is the outermost container